.clang-format

```diff
@@ -1,221 +1,64 @@
-# This file is an example configuration for clang-format 5.0.
-#
-# Note that this style definition should only be understood as a hint
-# for writing new code. The rules are still work-in-progress and does
-# not yet exactly match the style we have in the existing code.
-
-# Use tabs whenever we need to fill whitespace that spans at least from one tab
-# stop to the next one.
-#
-# These settings are mirrored in .editorconfig.  Keep them in sync.
-UseTab: Always
-TabWidth: 8
-IndentWidth: 8
-ContinuationIndentWidth: 8
-ColumnLimit: 80
-
-# C Language specifics
+# 0 "../cppMakeProject/.clang-format"
+# 0 "<built-in>"
+# 0 "<command-line>"
+# 1 "/usr/include/stdc-predef.h" 1 3 4
+# 0 "<command-line>" 2
+# 1 "../cppMakeProject/.clang-format"
```

Comment:
It's probably an idea in need of improvement to change the `.clang-format` file in a commit that claims to reformat the header files...